policy/modules/contrib/boothd.te

```diff
@@ -0,0 +1,82 @@
+policy_module(boothd, 1.0.0)
+
+########################################
+#
+# Declarations
+#
+
+type boothd_t;
+type boothd_exec_t;
+init_daemon_domain(boothd_t, boothd_exec_t)
+
+type boothd_etc_t;
+files_config_file(boothd_etc_t)
+
+type boothd_unit_file_t;
+systemd_unit_file(boothd_unit_file_t)
+
+type boothd_var_run_t;
+files_pid_file(boothd_var_run_t)
+
+type boothd_var_t;
+files_type(boothd_var_t)
+
+########################################
+#
+# boothd local policy
+#
+allow boothd_t self:fifo_file rw_fifo_file_perms;
+allow boothd_t self:capability { chown dac_override dac_read_search net_admin setgid setuid sys_nice sys_resource };
+allow boothd_t self:netlink_route_socket create_netlink_socket_perms;
+allow boothd_t self:process { setsched setrlimit };
+allow boothd_t self:tcp_socket create_stream_socket_perms;
+allow boothd_t self:udp_socket create_stream_socket_perms;
+allow boothd_t self:unix_dgram_socket create_socket_perms;
+allow boothd_t self:unix_stream_socket create_stream_socket_perms;
+
+read_files_pattern(boothd_t, boothd_etc_t, boothd_etc_t)
+
+manage_dirs_pattern(boothd_t, boothd_var_run_t, boothd_var_run_t)
+manage_files_pattern(boothd_t, boothd_var_run_t, boothd_var_run_t)
+files_pid_filetrans(boothd_t, boothd_var_run_t, { dir file} )
+
+manage_dirs_pattern(boothd_t, boothd_var_t, boothd_var_t)
+files_var_filetrans(boothd_t, boothd_var_t, dir, "booth")
```

Comment:
`files_var_lib_filetrans()` would be more appropriate, but is this needed at all? The directory is in the package.